kyo-prelude/shared/src/main/scala/kyo/Stream.scala

```diff
@@ -660,10 +661,23 @@ sealed abstract class Stream[V, -S] extends Serializable:
         )
     end splitAt
 
+    /** Transform with a [[Pipe]] of corresponding input streaming element type.
+      *
+      * @see
+      *   [[kyo.Pipe.transform]]
+      *
+      * @param sink
+      *   Pipe to transform stream with
+      * @return
+      *   A new stream of transformed element type `A`
+      */
+    def into[A, S2](pipe: Pipe[V, A, S2])(using Tag[V], Frame): Stream[A, S & S2] =
```

Comment:
how about `map` instead of `into`? `Pipe` is similar to a `map` function

Reply:
It's similar but different in important ways. `map` makes it seem like each element will be transformed, which is not necessarily the case. The input and output streams can be entirely different lengths, for instance.